src/Drawables/DrawableText.vala

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) 2019-2021 Alecaddd (https://alecaddd.com)
```

Comment:
You probably want to put your information here

Reply:
Nah, the copyright of every file is correct, then each author can write their name blow it in the Authored section.
Only change here is that this should only have 2021 since this file was created now.